include/otf2xx/traits/reference.hpp

```diff
@@ -76,12 +76,6 @@ namespace traits
     {
     };
 
-    template <typename Def, otf2::common::group_type Type>
```

Comment:
I suggested using 2 traits (one being trivial) so that you still can do `reference_type<Def>` generically. This might be required for generic code and doesn't cost much (rename `reference_type` to something with `tag` [`tag_type`?] and add 2 lines for the new `reference_type`)

Reply:
Use `otf2::reference<Def>::ref_type` or `otf2::traits::reference_type<typename Def::tag_type>::type` instead.

Reply:
Hm... ok. Could you add a comment that this trait should only be specialized for base/tag types? On that matter: Why is the trait not using the OTF2 types? If an OTF2 type would appear twice the mistake will be obvious and the test for that trait will be superflous.